src/api/app/controllers/search_controller.rb

```diff
@@ -71,13 +71,26 @@ def owner_group_or_user
     end
   end
 
-  def owner_package_or_project
-    return if params[:project].blank?
+  def owner_packages_or_projects
+    return if params[:project].blank? and params[:package].blank?
```

Comment:
```suggestion
    return if params[:project].blank? && params[:package].blank?
```

Please, do not use the `and` as a boolean operator. The `and` and `&&` are not exactly the same, the first one is used for flow control not for boolean logic... You can find more info [here](https://avdi.codes/using-and-and-or-in-ruby/)